packages/components/Atoms/Global/scss/tools/_svg.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rpl_svg_stroke($color, $size, $linecap: false, $dasharray: false, $dashoffset: false) {
```

Comment:
Can we have a comment to explain how and when to use this?
Also, I feel we need a default value for color and size.

Reply:
Added a default (black and 1px) with comment.